backend/src/services/kms/kms-service.ts

```diff
@@ -66,22 +66,55 @@ export const kmsServiceFactory = ({
 }: TKmsServiceFactoryDep) => {
   let ROOT_ENCRYPTION_KEY = Buffer.alloc(0);
 
+  const getRootEncryptionKey = (encryptionAlgorithm: SymmetricEncryption) => {
```

Comment:
Put a dollar symbol to indicate it is internal only like `$getRootEncryptionKey`

Reply:
refactored